ReactiveCocoaTests/Swift/TestError.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  TestError.swift
+//  ReactiveCocoa
+//
+//  Created by Almas Sapargali on 1/26/15.
+//  Copyright (c) 2015 GitHub. All rights reserved.
+//
+
+import Foundation
+import ReactiveCocoa
+
+enum TestError {
+	case Empty
+	case WithCode(Int)
```

Comment:
We could probably just start with a `Default` error for now. We don't need multiple options—the main goal is just getting away from `NSError`.